CMakeLists.txt

```diff
@@ -14,6 +14,11 @@
 #     CMake files. It is related to ZLIB support which is planned to be removed.
 #     When the support is removed, the associated include_directories command
 #     will be removed as well as this note.
+#   + CMake targets are designed to be able to be altered by calling CMake in
```

Comment:
This is not related to "Include directories" thus should be in another section. Thus something like:
` # - MBEDTLS_TARGET_PREFIX: ...`

Otherwise just document MBEDTLS_TARGET_PREFIX here not the "mbedtls_target", "mbedcrypto_target" ... "internal" variables.
Something similar to what you have done for the change log will be ok.

Reply:
Done